README.md

```diff
@@ -33,27 +33,29 @@ lit-html — an efficient, expressive, extensible HTML templating library for Ja
 
 Sorted by creation date (oldest first).
 
-- [@polymer/lit-element](https://github.com/Polymer/lit-element)
-- [gluonjs](https://github.com/ruphin/gluonjs)
-- [lit-element](https://github.com/DiiLord/lit-element)
-- [lit-html-element](https://github.com/kenchris/lit-element)
-- [fit-html](https://github.com/Festify/fit-html)
-- [lit-html-brackets](https://github.com/bgotink/lit-html-brackets)
-- [fabys-lit-element](https://github.com/fabysdev/fabys-lit-element)
-- [@littleq/element-lite](https://github.com/tjmonsi/element-lite)
-- [@corpuscule/element](https://github.com/corpusculejs/corpuscule)
-- [@dabolus/localized-lit-element](https://github.com/Dabolus/localized-lit-element)
-- [lit-apollo](https://github.com/bennypowers/lit-apollo)
+- [@polymer/lit-element](https://github.com/Polymer/lit-element), an ultra-light custom element base class with a simple but expressive API.
+- [gluonjs](https://github.com/ruphin/gluonjs), a lightweight library for building Web Components and applications.
+- [lit-element](https://github.com/DiiLord/lit-element), implements lit-html via a LitElement class.
+- [lit-html-element](https://github.com/kenchris/lit-element), a base class for creating Web Components using lit-html.
+- [fit-html](https://github.com/Festify/fit-html), fit-html is a combination of lit-html, Web Components and Redux.
+- [lit-html-brackets](https://github.com/bgotink/lit-html-brackets), lit-html extension that uses a bracket syntax similar to angular's template syntax.
+- [fabys-lit-element](https://github.com/fabysdev/fabys-lit-element), a base class for creating ES6 Web Components using TypeScript and lit-html.
```

Comment:
please remove "ES6", although repo description contains it. This is not necessary, and other descriptions do not contain it so better to be aligned with these